src/webview/webview2/mod.rs

```diff
@@ -616,7 +616,7 @@ window.addEventListener('mousemove', (e) => window.chrome.webview.postMessage('_
           });
         }
 
-        win32wm::WM_SETFOCUS => {
+        win32wm::WM_ENTERSIZEMOVE | win32wm::WM_SETFOCUS => {
```

Comment:
This will focus run only once when the resize/move starts, is it better to use `WM_EXITSIZEMOVE`?

Reply:
No it needs to run when it starts moving, otherwise the window will move but the drop-down menu will stay in it's current position.

Reply:
I just tried both and it doesn't matter both failed. When I start moving the window, the webview is focused and it will close the select dropdown but when I click on it, the drop down appears in the previous location. What version of webview2 are you on?

Reply:
105, but for me it only opened in the old location once really randomly.
And to be clear, this PR is to close the drop-down automatically. After that it's up to the webview to move the actual drop-down position (which MS mostly fixed) - so it's more about the second issue i linked

Reply:
Oh btw i only tested this in a tauri app, not in a wry example

Reply:
ah damn i can repro it. When the dropdown is already closed before moving it opens at the wrong location. if it was open and auto-closed it works as expected.

Reply:
I am on 105 as well and the only difference this PR made is that it closes the dropdown but the location bug still exists

Reply:
I still see the bug even when it is open and gets auto-closed

Reply:
guess i tested it in a really weird way then idk. It also seems to fix its position on hover change which is kinda funny and probably what tricked me here idk.

Reply:
when it comes to Windows and Microsoft, everything is funny